### ISSUE TITLE
Allow farmer to view results from all nodes in field

### ISSUE THREAD

**Issue by ARR8** [ARR8/soil-analysis-pgbuild#12]

**Comment by ARR8** [comment]
Possible idea: maps view?

**Comment by ARR8** [comment]
Would be platform-dependent. Google maps for android, apple maps for iOS, bing maps for WP. There's probably a phonegap plugin for maps

**Comment by benjamin-reichman** [comment]
I disagree with using apple maps unless if we have to, because of the quality of that software (unless if they fixed all the problems with the software)

**Comment by ARR8** [comment]
What I mean is we should use the native map interface for each platform. I don't really think it's our place to judge what software operating systems should use.

Of course, you do have a point, and we could always just use the Google Maps API. The problem with that is that it only seems to exist for Android and iOS.
